packages/app-store/zohocalendar/DESCRIPTION.md

```diff
@@ -0,0 +1,6 @@
+---
+items:
+  - /api/app-store/zohocalendar/ZCal1.jpg
```

Comment:
```suggestion
  - ZCal1.jpg
```
No need to use full URL. This should also work. It avoids hardcoding app slug and thus simplifies changing it(if needed)